app/controllers/observations_controller.rb

```diff
@@ -1,4 +1,4 @@
-class ProcedureObservationsController < ApplicationController
+class ObservationsController < ApplicationController
   before_action :authorized?
```

Comment:
Nice consolidation! It's convenient to have the set_experiment method to auto-find for us.